### DISCUSSION TITLE
@マークのエスケープはできますか？

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#236] [ヘルプ]
ご担当者　様

[Markdown記法 チートシート](https://qiita.com/Qiita/items/c686397e4a0f4f11683d)などを参照し、調べても解決方法が見つからなかったため、ご教示ください。

例として添付のキャプチャのように記事の中で@をつけて表現したい場合、現在のQiitaの仕様だと@に続けて文字列を記述するとユーザー名のリンクになってしまいます。

Code blocksなどでは問題ないと思いますが、シンプルに記事中で@をリンクなしで表現したい場合はどのように記述すればでよいでしょうか？
![スクリーンショット 2022-06-08 23 30 05](https://user-images.githubusercontent.com/38848514/172643373-64500e29-1be7-40ee-81e0-1b9d637678d0.png)

**Comment by tanaka-asami415** [comment]
@Brutus03 
お問い合わせいただきありがとうございます🙏

大変申し訳ありません。現状は「@をリンクなしで表現」することができません🙇💦
すぐに修正ができるか、一度確認してみます💪

**Reply by shiracamus** [reply]
バックスラッシュ＋@ で無効になるといいですね。
それまでは、シングルバッククオートで囲って回避かな。

**Reply by tanaka-asami415** [reply]
@Brutus03 @shiracamus 
調査したところ、修正はすぐにはできなさそうです🙇💦
ただ、できるようにはしたい:muscle:と思っていますので、お待ちいただけると幸いです🙏

**Comment by Phroneris** [comment]
![markdown_at](https://user-images.githubusercontent.com/9891867/177297219-d92b8e16-d45c-4dc6-8a4d-20d05234c766.png)

アットマークを `<span>` で囲ってみたらリンク化回避できました。
ひとまず閲覧やコピペには支障無いかと思います。

でもやっぱり、バックスラッシュか何かでスマートにエスケープできたら一番ですね…

**Reply by tanaka-asami415** [reply]
@Phroneris 
ありがとうございます🙏
>バックスラッシュか何かでスマートにエスケープできたら一番ですね…

はい！🙇何らかの方法で簡単にできるようにしたいと思っておりますので、お待ちいただけると幸いです🙇💦
